sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/DigitalTwinsRestClient.cs

```diff
@@ -55,37 +56,23 @@ internal async Task<Response<string>> AddAsync(string id, string twin, CreateDig
                 throw new ArgumentNullException(nameof(twin));
             }
 
```

Comment:
Do we not need this anymore? using DiagnosticScope scope = _clientDiagnostics.CreateScope("DigitalTwinsClient.Add");

Reply:
If we have non-trivial custom code, we should still need it.

Reply:
I believe during the preview timeframe, diagnostic scoping was generated by default. So when we copied code from the generated folder to the Customized folder, we got scoping. But I believe that's no longer the case. It's up to the client library authors to do so. I re-added the scope here. I will create a task to go through all our APIs and ensure scoping is added when appropriate.